unit_tests/src/test_file_object.cpp

```diff
@@ -0,0 +1,16 @@
+#include <catch2/catch.hpp>
+
+#include "irods/irods_file_object.hpp"
+
+TEST_CASE("file_object")
+{
+    SECTION("test_file_object_size_can_be_negative__issue_7154")
```

Comment:
The the C++ unit tests, we don't include underscores to separate words.

Compare your section title to other unit tests and try to make it align with how those are worded.